go/src/koding/kites/config/config.go

```diff
@@ -104,6 +104,36 @@ func (u *URL) WithPath(paths ...string) *URL {
 	return &URL{URL: &ur}
 }
 
+// StringOrBool implements json.Unmarshaler, which can parse
+// either boolean type or converts string to bool.
+type StringOrBool bool
+
+var _ json.Unmarshaler = (*StringOrBool)(nil)
+
+func (b *StringOrBool) UnmarshalJSON(p []byte) error {
```

Comment:
It's not, input can be either `[]byte("true")` (boolean) or `[]byte("\"true\"")` (string), thus the unmarshalling. Alternatively I can use strconv.Unquote which could spare two lines or three, up to you :-)